tests/test_local_db.py

```diff
@@ -56,6 +56,22 @@ def test_local_storage_model(mocker, database_with_model, second_draft_of_model)
     assert overwritten_model.flavor == "pytorch"
 
 
+def test_local_db_clone(mocker, garden_client, garden_all_fields, tmp_path):
+    # mock to replace "~/.garden/db"
+    mocker.patch("garden_ai.local_data.LOCAL_STORAGE", new=tmp_path)
+
+    # mock fetch and new garden creation operations
+    mocker.patch(
+        "garden_ai.client.GardenClient.get_published_garden",
+        return_value=PublishedGarden.from_garden(garden_all_fields),
+    )
+    mocker.patch("garden_ai.client.GardenClient.create_garden", return_value=None)
+
+    garden_client.clone_garden(garden_all_fields.doi, silent=True)
+
```

Comment:
should add a quick comment here explaining the behavior we're asserting, i.e. cloning a published garden should rehydrate local storage with the pipeline metadata copied from the garden